testing/pom.xml

```diff
@@ -14,13 +14,42 @@
     <packaging>jar</packaging>
 
     <properties>
-        <docker-client.version>8.16.0</docker-client.version>
-        <jnr-ffi.version>2.2.12</jnr-ffi.version>
         <pact.version>3.6.15</pact.version>
-        <hamcrest.version>2.2</hamcrest.version>
     </properties>
 
     <dependencies>
+        <!-- Main dependencies that are imported from one of the BOMs specified
+             in <dependencyManagement> in the parent pom, so no explicit versions needed -->
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+        </dependency>
+        <!-- we need to override the vintage engine's junit scope for version  >= 4.13 that restricts usage to test -->
```

Comment:
I don’t fully understand what this comment means…

Reply:
So, some of the code in `testing`, for example in `PostgresDockerRule`, is using imports like `org.junit.rules.*`, `junit.framework.*`, and `org.junit.runner.*`. In DW 2 it was using junit:junit 4.12. 4.12 had a scope of `compile`. It was changed in 4.13 to `test`. Since some of our code uses junit imports in non-test code, our imports fail.